common/src/main/java/io/homeassistant/companion/android/common/data/integration/Entity.kt

```diff
@@ -631,15 +631,22 @@ val <T> Entity<T>.friendlyName: String
 
 fun <T> Entity<T>.friendlyState(context: Context, options: EntityRegistryOptions? = null, appendUnitOfMeasurement: Boolean = false): String {
     var friendlyState = when (state) {
+        "armed_away" -> context.getString(commonR.string.state_armed_away)
```

Comment:
Missing states: [`pending`, `arming`, `disarming`](https://github.com/home-assistant/core/blob/dev/homeassistant/components/alarm_control_panel/strings.json#L32-L44) (those last two should probably be added to the executing states as well)

Reply:
ah good catch!